packages/fxa-settings/src/lib/gql.ts

```diff
@@ -88,11 +88,6 @@ export const errorHandler: ErrorHandler = ({ graphQLErrors, networkError }) => {
           // Redirect to /signin since that page will send the user
           // to correct location
           return window.location.replace(`/signin?action=email&service=sync`);
-        } else if (
-          window.location &&
-          window.location.pathname.includes('signin_token_code')
-        ) {
-          // If the user is already on the signin_token_code page, we don't need to redirect
```

Comment:
Was this appearing to fix the problem locally? I'm a bit confused how since I think it's just saying "don't redirect if on this page" right? 🤔

Reply:
I was getting the `Must verify` issue if had 2FA on and manually navigated to `signin_token_page`. This seemed to fix that problem and didn't appear to impact anything else? We have changed how the `signin_totp_page` gets its session token, so perhaps the issue this was solving wasn't happening anymore.

Reply:
> if had 2FA on and manually navigated to signin_token_page

Ahh OK, that makes sense. I actually wonder if we want to replace all session token references to just `sessionToken` instead of keeping it in signin state which would help with this...